core/domain/html_cleaner.py

```diff
@@ -405,7 +450,38 @@ def convert_to_text_angular(html_data):
     # html strings they are stored as <br>. Since both of these
     # should match and <br> and <br/> have same working,
     # so the tag has to be replaced in this way.
-    return str(soup).replace('<br/>', '<br>')
+    return unicode(soup).replace('<br/>', '<br>')
```

Comment:
Ditto, I don't think you need the `unicode` cast. Did you test with unicode chars?

Reply:
Answered!